service/devdoc/registry_requirements.md

```diff
@@ -628,6 +628,7 @@ Request-Id: <guid>
 DELETE /devices/<encodeURIComponent(deviceId)>/modules/<encodeURIComponent(moduleId)>?api-version=<version> HTTP/1.1
 Authorization: <sharedAccessSignature>
 Request-Id: <guid>
+If-Match: "*"
```

Comment:
This is brutal. it's an error we made on device identity registry early on, but we should respect/enable the use of etags.